sdk/metricsadvisor/Azure.AI.MetricsAdvisor/CHANGELOG.md

```diff
@@ -39,6 +45,8 @@
 - In `DataPointAnomaly`, renamed property `AnomalyDetectionConfigurationId` to `DetectionConfigurationId`.
 - In `DataFeedMetric`, renamed constructor parameter `metricName` to `name` only.
 - In `DataFeedDimension`, renamed constructor parameter `dimensionName` to `name` only.
+- In `MetricAnomalyAlertScope`, renamed static methods `GetScopeFor<...>` to `CreateScopeFor<...>`. For instance, `GetScopeForSeriesGroup` was renamed to `CreateScopeForSeriesGroup`.
+- Changed signature of the `MetricAnomalyAlertScope.GetScopeForTopNGroup` method to take the parameters `top`, `period`, and `minimumTopCount` directly. For this reason, removed the public constructor of `TopNGroupScope`.
```

Comment:
This one is for convenience. Originally we needed to do:
```cs
var groupScope = new TopNGroupScope(1, 2, 3);
var alertScope = MetricAnomalyAlertScope.GetScopeForTopNGroup(groupScope);
```
Now:
```cs
var alertScope = MetricAnomalyAlertScope.GetScopeForTopNGroup(1, 2, 3);
```

The `TopNGroupScope` class is not used anywhere in the code, so this makes it more convenient.